src/plugins/vis_types/vislib/public/vislib/components/tooltip/_tooltip.scss

```
@@ -49,8 +49,7 @@
 }
 
 .visTooltip__headerIcon {
-  flex: 0 0 auto;
-  padding-right: $euiSizeS;
+  padding-right: $euiSizeXS;
```

```suggestion
  margin-right: $euiSizeXS;
```

It's safer to add outside margin than inside padding to an EUI component 😉

---

Oh nice!! good to know!